src/main/java/com/avengers/gamera/service/LikeService.java

```diff
@@ -54,6 +54,7 @@ public void createLike(Long articleId) {
         userRepository.save(user);
         article.getLikeUsers().add(user);
         article.setLikeUsers(article.getLikeUsers());
+        article.setLikeNum(article.getLikeNum()+1);
```

Comment:
反正都联级查询了  最好类似这样  用真实的数据个数来计算
```suggestion
        article.setLikeNum(article.getLikeUsers().size + 1);
```

如果原来的 getLikeNum 本来就是错误的 那这里直接+1 就会一直错下去